src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
The `div` elements are not semantic. It would be better if you used a semantic element like `section` or `article` for the `stars` containers, and an ordered list `ol` for the stars themselves, because the number of filled stars has a meaning. It would improve your page SEO and help screen readers.